lib/config/options/index.ts

```diff
@@ -1752,6 +1752,12 @@ const options: RenovateOptions[] = [
     default: 'auto',
   },
   // PR Behaviour
+  {
+    name: 'keepUpdatedLabel',
+    description:
+      'Label to request Renovate bot always rebase to keep branch updated.',
+    type: 'string',
+  },
```

Comment:
Please add `supportedPlatforms` field (should be any which supports labels)

Reply:
:+1: , I copied the supported platforms from `stopUpdatingLabel`.